backend/databasemodel/model.sql

```diff
@@ -1542,9 +1563,12 @@ CREATE TABLE kooste.museovirastoarcrest_muinaisjaannokset (
 	luontipvm date,
 	muutospvm date,
 	www text,
+	deleted boolean NOT NULL DEFAULT false,
 	CONSTRAINT museovirastoarcrest_muinaisjaannokset_pk PRIMARY KEY (mjtunnus)
 
 );
+CREATE INDEX ON kooste.museovirastoarcrest_muinaisjaannokset (deleted);
+CREATE INDEX ON kooste.tamperewfs_luonnonmuistomerkit (visibility);
```

Comment:
I think there is wrong table name here (kooste.tamperewfs_luonnonmuistomerkit -> kooste.museovirastoarcrest_muinaisjaannokset) :)